shared/constants/chat2/convostate.tsx

```diff
@@ -316,6 +311,7 @@ export type ConvoState = ConvoStore & {
   }
   getExplodingMode: () => number
   getEditInfo: () => {exploded: boolean; ordinal: T.Chat.Ordinal; text: string} | undefined
+  isMetaGood: () => boolean
```

Comment:
use this helper instead of explicitly checking `getStore(convoid).meta.id === convoid` everywhere